static/app/views/userFeedback/index.spec.jsx

```diff
@@ -143,18 +144,23 @@ describe('UserFeedback', function () {
       organization: TestStubs.Organization({
         projects: [TestStubs.Project({isMember: true})],
       }),
-      location: {query: {project: ['112', '113']}, search: ''},
+      location: {pathname: 'sentry', query: {project: ['112', '113']}, search: ''},
       params: {
         orgId: organization.slug,
       },
     };
-    const wrapper = mountWithTheme(
+    render(
       <OrganizationContext.Provider value={organization}>
         <UserFeedback {...params} />
       </OrganizationContext.Provider>,
-      routerContext
+      {context: routerContext}
     );
 
-    expect(wrapper.find('UserFeedbackEmpty').prop('projectIds')).toEqual(['112', '113']);
+    expect(screen.getByTestId('user-feedback-empty')).toBeInTheDocument();
```

Comment:
should we check for `screen.getByText('What do users think?')` instead of using a `data-test-id`?

Reply:
I considered it, but I didn't want to have to update the test if we changed that text.